modules/cli-about-cli.adoc

```diff
@@ -6,21 +6,10 @@
 [id="cli-about-cli_{context}"]
 = About the OpenShift CLI
 
-With the {oc-first}, you can create applications and manage {product-title}
-ifdef::openshift-rosa[]
-(ROSA)
-endif::openshift-rosa[]
-projects from a terminal. The OpenShift CLI is ideal in the following situations:
+With the {oc-first}, you can create applications and manage {product-title} projects from a terminal. The OpenShift CLI is ideal in the following situations:
 
 * Working directly with project source code
```

Comment:
suggestion to use punctuation in list (period/full stop at end of each bullet)
Example 

With the {oc-first}, you can create applications and manage {product-title} projects from a terminal. The OpenShift CLI is ideal in the following situations:

* Working directly with project source code.